src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/util/TestStorageHelper.java

```diff
@@ -0,0 +1,109 @@
+package uk.gov.hmcts.reform.bulkscanprocessor.util;
+
+import com.microsoft.azure.storage.CloudStorageAccount;
+import com.microsoft.azure.storage.StorageException;
+import com.microsoft.azure.storage.blob.CloudBlobClient;
+import com.microsoft.azure.storage.blob.CloudBlobContainer;
+import com.microsoft.azure.storage.blob.CloudBlockBlob;
+import org.testcontainers.containers.DockerComposeContainer;
+
+import java.io.File;
+import java.net.URISyntaxException;
+import java.security.InvalidKeyException;
+
+import static uk.gov.hmcts.reform.bulkscanprocessor.helper.DirectoryZipper.zipDir;
+
+public class TestStorageHelper {
+
+    private static TestStorageHelper INSTANCE;
+
+    public static final String CONTAINER_NAME = "bulkscan";
+    public static final String ZIP_FILE_NAME = "1_24-06-2018-00-00-00.zip";
+
+    private static DockerComposeContainer<?> dockerContainer;
+    public static CloudBlobClient cloudBlobClient;
+    private CloudBlobContainer testContainer;
+
+    private TestStorageHelper() {
+        // empty constructor
+    }
+
+    public static TestStorageHelper getInstance() {
+        if (INSTANCE == null) {
+            INSTANCE = new TestStorageHelper();
+        }
+
+        return INSTANCE;
+    }
+
+    public static void createDocker() {
+        dockerContainer = new DockerComposeContainer<>(
+            new File("src/integrationTest/resources/docker-compose.yml")
+        ).withExposedService("azure-storage", 10000);
+    }
+
+    public static void startDocker() {
+        if (dockerContainer == null) {
+            throw new RuntimeException(
+                "Cannot start container. Did you create one with "
+                    + TestStorageHelper.class.getSimpleName()
+                    + "::createDocker in a static context?"
+            );
+        }
+
+        try {
+            dockerContainer.start();
+            cloudBlobClient = CloudStorageAccount
```

Comment:
method says start docker but doing more than just starting :)

Reply:
this should be outside or method should be something else

Reply:
Well. It is "StorageHelper" afterall. Could call `setUp` but it's just too generic I guess. I don't mind what it is now but if you have something specific in mind - let me know ;)

Reply:
I think it's better to have method like `initialise` or `setUp` where you can club both `createDocker` and `startDocker` and get rid of below 
```
if (dockerContainer == null) {
             throw new RuntimeException(
                 "Cannot start container. Did you create one with "
                     + TestStorageHelper.class.getSimpleName()
                     + "::createDocker in a static context?"
             );
         }
```

Reply:
was random suggestion ^ ;). with if statement.

plus, it's generic class. helper. setup sounds more test related. i'll rename `initialiseStorage` ;)